pkg/hooks/hooks_test.go

```diff
@@ -77,3 +79,44 @@ func testManifest() pods.Manifest {
 	builder.SetID(podId)
 	return builder.GetManifest()
 }
+
+func TestHookRunWithTimeout(t *testing.T) {
```

Comment:
The case gets exercised above in `TestExecutableHooksAreRun`, so I left it out here.

I didn't refresh before merging, so I just saw this.  I don't mind adding it back in a quick PR if you like.